<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sensor Fusion Calibrate</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="serialLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialList">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="baudLabel">
        <property name="text">
         <string>Baud</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudList">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="serialOpenCloseButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {color: green;}</string>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="calibration">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QGroupBox" name="offsetGroupBox">
          <property name="title">
           <string>Offset</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="offsetLineEdit1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="offsetLineEdit2"/>
           </item>
           <item>
            <widget class="QLineEdit" name="offsetLineEdit3"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="mappingGroupBox">
          <property name="title">
           <string>Mapping</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineEdit_8"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_9"/>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEdit_10"/>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_11"/>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="lineEdit_12"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="visual">
       <attribute name="title">
        <string>Visual</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="data">
       <attribute name="title">
        <string>Collected Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QTableWidget" name="dataTableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="terminal">
       <attribute name="title">
        <string>Serial Terminal</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="SerialTerminal" name="terminalPlainTextEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="terminalScrollCheckbox">
            <property name="text">
             <string>Scroll to bottom</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialTerminal</class>
   <extends>QPlainTextEdit</extends>
   <header>terminal.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
